docs/source/stability_policy.rst

```diff
@@ -0,0 +1,132 @@
+
+PTB has dependencies to 3rd party packages.
+The versions that PTB uses of these 3rd party packages may change if that does not change PTBs public API.
+
+PTB does not give guarantees about which Python versions are supported.
+In general, we will try to support all Python versions that have not yet reached their end of life, but we reserve ourselves the option to drop support for Python versions earlier if that benefits the advancement of the library.
+
+.. _bot-api-functionality-1:
+
+Bot API Functionality
+~~~~~~~~~~~~~~~~~~~~~
+
+Comparison of equality of the classe in the :mod:`telegram` package is subject to change and the PTB team will update the behavior to best reflect updates in the Bot API.
+
+When arguments of the Bot API methods change order or become optional/mandatory due to changes in the Bot API, PTB will always try to reflect these changes.
+While we try to make such changes backwards compatible, this is not always possible or only with significant effort.
+In such cases we will find a trade-off between backwards compatibility and fully complying with the Bot API, which may result in breaking changes.
+We highly recommend using keyword arguments, which can help to make such changes non-breaking on your end.
+
+When the Bot API changes attributes of classes, the method :meth:`telegram.TelegramObject.to_dict` will change as necessary to reflect these changes.
+In particular, attributes deprecated by Telegram will be removed from the returned dictionary.
+Deprecated attributes that are still passed by Telegram will be available in the :attr:`~telegram.TelegramObject.api_kwargs` dictionary as long as PTB can support that with feasible effort.
+Since attributes of the classes in the :mod:`telegram` package are not writable, we may change them to properties where appropriate.
+
+Development Versions
+~~~~~~~~~~~~~~~~~~~~
+
+Before a feature is in a release, it is not covered by this policy and may change.
+Pre-Release marked as alpha, beta or release candidate are not covered by this policy either.
+
+Security
+~~~~~~~~
+
+We make exceptions from our stability policy for security.
+We will violate this policy as necessary in order to resolve a security issue or harden PTB against a possible attack.
+
+Versioning
+----------
+
+PTB uses a versioning scheme that roughly follows `https://semver.org/ <https://semver.org/>`_, although it may not be quite as strict.
+
+Given a version of PTB X.Y.Z,
+
+-  X indicates the major version number.
+   This is incremented when backwards incompatible changes are introduced.
+-  Y indicates the minor version number.
+   This is incremented when new functionality or backwards compatible changes are introduced by PTB.
+   *This is also incremented when PTB adds support for a new Bot API version, which may include backwards incompatible changes in some cases as outlined* :ref:`below <bot-api-versioning>`.
```

Comment:
```suggestion
   *This is also incremented when PTB adds support for a new Bot API version, which may include small backwards incompatible changes in some cases as outlined* :ref:`below <bot-api-versioning>`.
```

Reply:
Here I don't the see added value either as the word "small" is not very objective

Reply:
Hmm, I still prefer the word flow more, but I can understand your sentiment